docs/en/Security.md

```diff
@@ -18,7 +18,7 @@ encryption.
 POSIX permission model is used in Alluxio to assign permissions and
 control access rights.
 
-By default Alluxio runs in non-secure mode in which no authentication and authorization are
+By default Alluxio runs in SIMPLE secure mode in which authentication and authorization are
```

Comment:
done, I've added some explanation. I plan to further improve security doc later. This PR is aimed for enable SIMPLE by default and fixing tests.